.ci/run_tests.sh

```diff
@@ -22,4 +22,6 @@ run_oldtests
 
 install_nvim
 
+run_single_includes_tests
```

Comment:
can this go directly after `build_nvim`, to save time? (If there is a problem, the PR author will know more quickly)